tests/playwright/src/specs/proxy-smoke.spec.ts

```diff
@@ -0,0 +1,147 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { ProxyTypes } from '../model/core/types';
+import { ProxyPage } from '../model/pages/proxy-page';
+import { expect as playExpect, test } from '../utility/fixtures';
+import { handleConfirmationDialog } from '../utility/operations';
+
+const invalidProxyUrl = 'invalid-proxy-url';
+const httpProxyUrl = 'http://http.proxy:8080';
+const httpsProxyUrl = 'https://http.proxy:8443';
+const hostsDomains = 'localhost';
+
+let proxyPage: ProxyPage;
+
+test.beforeAll(async ({ page, runner, welcomePage }) => {
+  runner.setVideoAndTraceName('proxy-e2e');
+  await welcomePage.handleWelcomePage(true);
+  proxyPage = new ProxyPage(page);
+});
+
+test.afterAll(async ({ navigationBar, page, runner }) => {
+  await navigationBar.openDashboard();
+  const settingsBar = await navigationBar.openSettings();
+  await settingsBar.proxyTab.click();
+  await playExpect(proxyPage.heading).toBeVisible();
+  await proxyPage.selectProxy(ProxyTypes.System);
+  await proxyPage.updateButton.click();
+  await handleConfirmationDialog(page, 'Proxy Settings', true, 'OK');
+  await runner.close();
+});
+
+test.describe.serial('Proxy settings ', { tag: '@smoke' }, () => {
+  test.beforeEach(async ({ navigationBar }) => {
+    await navigationBar.openDashboard();
+    const settingsBar = await navigationBar.openSettings();
+    await settingsBar.proxyTab.click();
+    await playExpect(proxyPage.heading).toBeVisible();
+  });
+
+  test('Proxy page assets and System proxy setup', async () => {
+    await playExpect(proxyPage.toggleProxyButton).toBeVisible();
+    await playExpect(proxyPage.toggleProxyButton).toHaveText(ProxyTypes.System);
+    await playExpect(proxyPage.httpProxy).not.toBeEnabled();
+    await playExpect(proxyPage.httpsProxy).not.toBeEnabled();
+    await playExpect(proxyPage.noProxy).not.toBeEnabled();
+    await playExpect(proxyPage.updateButton).toBeEnabled();
+  });
+
+  test('Manual proxy setup, validation and update', async ({ page }) => {
+    await proxyPage.selectProxy(ProxyTypes.Manual);
+    await playExpect(proxyPage.httpProxy).toBeEnabled();
+    await playExpect(proxyPage.httpsProxy).toBeEnabled();
+    await playExpect(proxyPage.noProxy).toBeEnabled();
+    // validation for http proxy input
+    await proxyPage.httpProxy.fill('invalid-proxy-url');
```

Comment:
```suggestion
    await proxyPage.httpProxy.fill(invalidProxyUrl);
```

Why are you using the literal string here to perform the fill and not using the invalidProxyUrl constant?